tests/config/c/nl_collapse_empty_body_functions.cfg

```diff
@@ -0,0 +1,2 @@
+nl_collapse_empty_body = true
+nl_collapse_empty_body_functions = true
```

Comment:
Based on the file tests/expected/c/00156-empty-braces.c, shouldn't this be 'false'? This would also match with the missing ! in the previous comment.

Second note: we should have two tests, one with:
```
nl_collapse_empty_body = true
nl_collapse_empty_body_functions = true
```
and one with 
```
nl_collapse_empty_body = true
nl_collapse_empty_body_functions = false
```
so we test the new option in both positive and negative case. One case should also collapse the function bodies, the other shouldn't.

Reply:
Nice, thanks to your new code I realized we were missing 2 test cases more.
We should have all 4 combinations of ```nl_collapse_empty_body``` and ```nl_collapse_empty_body_functions```, that is:
- true, true
- true, false
- false, true
- false, false
This will cover all the possible scenarios of the two options.